target/linux/ath79/dts/qca9533_tplink_cpe220-v3.dts

```diff
@@ -0,0 +1,151 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "qca953x.dtsi"
```

Comment:
One should create a common dtsi from this one and qca9533_tplink_cpe210.dtsi.

The only differences are eth1 and the lan leds. I would call the parent dtsi
qca9533_tplink_cpe.dtsi
or
qca9533_tplink_cpexxx.dtsi